data_models/models.py

```diff
@@ -127,6 +129,13 @@ class Meta:
     content = models.TextField('Brødtekst')
     deleted = models.BooleanField('Slettet', default=False)
 
+    episodes = models.ManyToManyField(
+        Episode,
+        blank=True,
+        verbose_name='Episoder',
+        help_text='Episoder blir sortert på program hvis artikkelen er lagret.'
```

Comment:
I don't understand the meaning of the help_text, perhaps "Legg til episoder som vises nederst i artikkelen"